tests/cli.rs

```diff
@@ -32,4 +65,22 @@ fn create_output_dir() {
     ])
     .assert()
     .failure();
+
+    let mut cmd = Command::cargo_bin("solang").unwrap();
+
+    cmd.args([
+        "compile",
+        "examples/flipper.sol",
+        "--target",
+        "solana",
+        "--contract",
+        "flapper,flipper", // not flipper
```

Comment:
I'm confused by this comment

Reply:
:rofl: yes, that's is not very clear. How about "not just flipper"